src/aap_eda/api/views/activation.py

```diff
@@ -277,9 +282,34 @@ def disable(self, request, pk):
         activation = get_object_or_404(models.Activation, pk=pk)
 
         if activation.is_enabled:
+            # clear the job from rq if it's pending/running
+            job_id = activation.current_job_id
+
+            if job_id:
+                queue = get_queue(name="activation")
+                job = queue.fetch_job(job_id)
+                if job and job.get_status(refresh=True) in [
+                    JobStatus.QUEUED.value,
+                    JobStatus.STARTED.value,
+                    JobStatus.DEFERRED.value,
+                    JobStatus.SCHEDULED.value,
+                ]:
+                    job.delete(delete_dependents=True)
+                    logger.info(
+                        f"Job {job.id} is removed from the queue: {queue.name}"
```

Comment:
I think it would be useful if we print here the activation_id or activation_name